competitions/template/ansible/inv/prod/group_vars/all/secret.tmpl

```diff
@@ -0,0 +1,11 @@
+---
+# mysql has no root password on this server
+mysql_root_password:
+
+db_password: <PASSWORD>
+mediawiki_admin_password: <PASSWORD>
+mediawiki_mwlib_password: <PASSWORDWITHNOSPACES>
+mediawiki_csv2wiki_password: <PASSWORD>
+
+simplesaml_okta_metadata_name: <COMPEITIONNAME>
```

Comment:
1. typo :-)
2. underscore
3. I think this one is indeed "COMPETITION_NAME" not "USUALLY_COMPETITION_NAME", but wanted to check.